src/orm/BlockEntity.ts

```diff
@@ -21,6 +21,9 @@ export default class BlockEntity {
   @Column()
   timestamp: Date
 
+  @Column({ type: 'char', length: 51 })
```

Comment:
I think this change broke my local terra remote server :( When I restarted it, it threw this error:
```
postgres_1       | 2021-10-04 11:28:48.392 UTC [44] ERROR:  column "proposer" contains null values
postgres_1       | 2021-10-04 11:28:48.392 UTC [44] STATEMENT:  ALTER TABLE "public"."block" ADD "proposer" character(51) NOT NULL
```
we probably had to make this field nullable: true